### ISSUE TITLE
was this abandoned?

### ISSUE THREAD

**Issue by RedactedProfile** [Omnomhub/omnomhub#52]
Im just confused. Neo4J lists this project as an example of things built with it, but I see nothing in terms of logic or display anywhere...

**Comment by michellesanver** [comment]
This project was not abandoned, we started making an API for it, and have had a lot of conceptual phases, and I've been busy with several things but certainly didn't abandon it! 

Starting to have more time on my hands, so we will see what happens in the near future, anything specific you wanted to see from this project that you were looking for? :)

This is a pet project, and unfortunately it had to suffer when real life got in the way.

**Comment by RedactedProfile** [comment]
No one understands that notion better than I hahha so many pet projects by the wayside 

Actually I was curious specifically if i could contribute! Looking to improve my knowledge and skill with neo, but I honestly have no idea where to start with this one lol 

My wife would love a site like this I think

**Comment by WyriHaximus** [comment]
Same here. So many projects so little time. Would love to take this off the ground into as we intended when we started this. So your help is much appriciated :+1: !

**Comment by michellesanver** [comment]
Agreed! Let's do this thing, I really need this website for my cooking ;-)
